app/controllers/api/v1/events_controller.rb

```diff
@@ -1,5 +1,40 @@
-class Api::V1::EventsController < ApplicationController
+class Api::V1::EventsController < ApiController
   def show
     @event = Event.find(params[:id])
   end
+
+  def create
+    @event = Event.new(event_params)
```

Comment:
Ah yes. Brian, when we were talking about this yesterday somehow I forgot about strong_parameters. Definitely check that out.